.rubocop_todo.yml

```diff
@@ -64,32 +64,6 @@ RSpec/AnyInstance:
     - 'spec/requests/appropriate_bodies/claim_an_ect/register_ect_spec.rb'
     - 'spec/wizards/application_wizard_step_spec.rb'
 
-# Offense count: 2
-# This cop supports unsafe autocorrection (--autocorrect-all).
-RSpec/BeEq:
```

Comment:
Are there any plans to fix the offences in here? It would be nice to abide by the cops.

Reply:
I plan to work my way through them.

Reply:
Nice, I can help!

Reply:
Please, do. I thought we could do them in small relevant batches. 👍🏼